testng-browserstack-parallel.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Parallel">
+    <test name="BStackDemo chrome">
```

Comment:
lets have the testng-*.xml files in the resources/conf/runners directory, as per the diagram here: https://browserstack.atlassian.net/wiki/spaces/CE/pages/2091057195/Customer+Demo+Repositories#Implementation